ui/app/AppLayouts/Wallet/popups/SignTransactionModalBase.qml

```diff
@@ -14,13 +14,15 @@ import StatusQ.Popups 0.1
 import StatusQ.Popups.Dialog 0.1
 
 import shared.controls 1.0
+import shared.stores 1.0
 
 import utils 1.0
 
 StatusDialog {
     id: root
 
     required property int loginType // RootStore.loginType -> Constants.LoginType enum
+    required property var formatBigNumber// => (number:string, symbol:string, noSymbolOption:bool) {}
```

Comment:
It's not skipped, pls note the signature isn't (and can't due to the `required` param) enforced

Reply:
> It's not skipped, pls note the signature isn't (and can't due to the `required` param) enforced

Not sure if we are referring to the same thing.

I mean that e.g. in SwapApproveCapModal we have calls formatBigNumber(root.fromTokenAmount, root.fromTokenSymbol), effectively providing noSymbolOption, suggested to be bool, as undefined.

Reply:
Yeah and in the end, that (will) resolve to:
```js
    function formatBigNumber(number: string, symbol: string, noSymbolOption: bool) {
        if (!number)
            return "N/A"
        if (!symbol)
            symbol = root.currentCurrency
        let options = {}
        if (!!noSymbolOption)
            options = {noSymbol: true}
        return formatCurrencyAmount(parseFloat(number), symbol, options)
    }
```

Reply:
So even if you pass 2 parameters, no harm's done, no warning whatsoever

Reply:
Now those type annotations are completely ignored. My concern is it will stop working when at some point in the future they will be taken into account.